packages/jobs/lib/app.ts

```diff
@@ -16,20 +17,55 @@ try {
     logger.info(`🚀 service ready at http://localhost:${port}`);
     const processor = new Processor(orchestratorUrl);
 
+    // We are using a setTimeout because we don't want overlapping setInterval if the DB is down
+    let healthCheck: NodeJS.Timeout | undefined;
+    const check = async () => {
+        try {
+            await db.knex.raw('SELECT 1').timeout(1000);
```

Comment:
that should be in the `/health` endpoint logic

Reply:
please read PR desc ☺️